dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ORCIDHelper.java

```diff
@@ -121,6 +128,23 @@ public static HttpResponse postWorkString(String baseURL, String id, String work
         }
     }
 
+    /**
+     * This updates an existing ORCID work
+     */
+    public static HttpResponse putWorkString(String baseURL, String id, String workString, String token, String putCode) throws IOException {
+        HttpPut postRequest = new HttpPut(baseURL + id + "/work/" + putCode);
+        postRequest.addHeader("content-type", "application/vnd.orcid+xml");
```

Comment:
You could use javax.ws.rs.core.HttpHeaders for header names. Although we so far only use them in a test class.